reporting_app/static/datatable.js

```diff
@@ -92,6 +92,18 @@ var merge_multi_sources_keep_first = function(dt_config){
     return _merge_multi_sources(dt_config, merge_on_keep_first);
 }
 
+var test_exist = function(variable){
+    if ( variable instanceof Array ) {
+        return variable.length > 0;
+    }
+    return variable !== undefined && variable !== null && variable;
```

Comment:
For some reason, this returns `variable` if the checks pass, not a boolean - is this alright? Also, 0 and empty objects (`{}`) technically evaluate to true here.